library/stub-android/src/main/java/android/os/Handler.java

```diff
@@ -0,0 +1,15 @@
+/*
+ * ToYou
+ * Copyright © 2023 Aram Meem Company Limited. All Rights Reserved.
+ */
+
+package android.os;
+
+public class Handler {
```

Comment:
I think need to add comment for what this class is added.

Reply:
The whole module exists to hold simple java stubs to avoid android dependency in main library. It is used as `classpath` dependency there.

Reply:
I understand that, but after a while we'll think about what it's for